src/node/hooks/express/static.js

```diff
@@ -15,6 +19,9 @@ exports.expressCreateServer = (hookName, args, cb) => {
   // file-specific hacks for ace/require-kernel/etc.
   args.app.all('/static/:filename(*)', minify.minify);
 
+  expressDurations.minification = Date.now() - preMinification;
```

Comment:
This is going to be super tiny duration because no processing is done between `preMinification` and this line—the above is only registering callbacks to be called later.